src/sentry/integrations/msteams/utils.py

```diff
@@ -86,25 +84,6 @@ def send_incident_alert_notification(action, incident, metric_value, method):
         logger.info("rule.fail.msteams_post", extra={"error": str(e)})
 
 
-def get_identity(user, organization_id, integration_id):
```

Comment:
Was this just dead code?

Reply:
It got replaced with `get_identity_or_404()` in a different PR.